builder/variants/defaults-mac.sh

```diff
@@ -3,14 +3,18 @@ XCODE_MAJOR_VER="$(xcodebuild -version | grep 'Xcode' | awk '{print $2}' | cut -
 
 FF_CFLAGS+="-I"$FFBUILD_PREFIX"/include"
 FF_LDFLAGS+="-L"$FFBUILD_PREFIX"/lib"
-FF_CONFIGURE+=" --disable-libjack --disable-indev=jack --enable-neon --enable-runtime-cpudetect --enable-audiotoolbox --enable-videotoolbox"
+FF_CONFIGURE+=" --disable-libjack --disable-indev=jack --disable-libxcb --disable-xlib --enable-neon --enable-runtime-cpudetect --enable-audiotoolbox --enable-videotoolbox"
 FFBUILD_TARGET_FLAGS="--disable-shared --enable-static --pkg-config-flags=\"--static\" --enable-pthreads --cc=clang"
 FF_HOST_CFLAGS="-I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include -I"$FFBUILD_PREFIX"/include"
 FF_HOST_LDFLAGS=""
-if [ $XCODE_MAJOR_VER -ge 15 ]; then
-  FF_HOST_LDFLAGS+="-Wl,-ld_classic "
-  export LDFLAGS="-Wl,-ld_classic"
-fi
+
+# This is no longer needed for latest xcode.
```

Comment:
Can we have a specific version number here please

Reply:
I'm not sure because I really don't know.

Reply:
I changed that to the current version I'm using. Not the exact version of this change but at least some sort of version can be used as reference.